src/mscorlib/ref/Compat/mscorlib.cs

```diff
@@ -2601,6 +2607,8 @@ public partial class Exception : System.Runtime.Serialization.ISerializable
         public virtual string StackTrace { get { return default(string); } }
         public System.Reflection.MethodBase TargetSite { get { return default(System.Reflection.MethodBase); } }
         protected event System.EventHandler<System.Runtime.Serialization.SafeSerializationEventArgs> SerializeObjectState { add { } remove { } }
+        [System.Runtime.CompilerServices.FriendAccessAllowedAttribute]
+        internal void AddExceptionDataForRestrictedErrorInfo(string restrictedError, string restrictedErrorReference, string restrictedCapabilitySid, object restrictedErrorObject, bool hasrestrictedLanguageErrorObject = false) { }
```

Comment:
The desired state should be:

- System.Private.CoreLib exposes two kinds of types/methods: types/methods for public consumption, and types/methods for internal consumption by other framework libraries. Example of the latter is this internal method.
- mscorlib facade has forwarders for the types for public consumption only.

We took a shortcut during mscorlib to System.Private.CoreLib rename: All binaries in corefx still reference System.Private.CoreLib via mscorlib facade. To make this shortcut work we need to have forwarders in mscorlib for the internal types/methods well. This makes it impossible to have a working system without mscorlib façade, creates dependency cycles (e.g. System.Console -> System.Runtime -> mscorlib -> System.Console) that will get more complex as we add more stuff. Something we need to cleanup.